### PR TITLE
Fixed some problems with char input, added comments.

### PR DESCRIPTION
**Pull request by NotHexadecimal**
As described in the title.

### CODE REVIEW

File: Javascript/brainfuck.html [hunk 1]
```diff
@@ -24,17 +24,19 @@
         border-style: none;
         background-color: #ddd;
         width: 100%;
-        margin-bottom: 20px;
+        margin-bottom: 10px;
```
Comment (TheTipo01) [inline Javascript/brainfuck.html:27]: I like it better at 30px

**Review by TheTipo01 — CHANGES_REQUESTED** [review]

File: Javascript/brainfuck.html (hunk 1 above)
Comment (NotHexadecimal) [reply Javascript/brainfuck.html:27]: I tried it with 30px, il looks wrong, there is too much space between elements.

Comment (TheTipo01) [reply Javascript/brainfuck.html:27]: ¯\_(ツ)_/¯

**Review by TheTipo01 — APPROVED** [review]

File: Javascript/brainfuck.html [hunk 2]
```diff
@@ -115,7 +121,7 @@
               break;
             case ',':
               if (inputPointer < input.length) {
-                memory[memoryPointer] = input[inputPointer];
+                memory[memoryPointer] = input.charCodeAt(inputPointer);
```
Comment (techmccat) [inline Javascript/brainfuck.html:124]: JS scares me

**Review by techmccat — COMMENTED** [review]
Why does it even need a method to index stuff corectly

**Comment by NotHexadecimal** [comment]
It just does, I don't make the rules...

**Comment by TheTipo01** [comment]
Javascript
Bottom text

**Pull request merged**